test/test-functions

```diff
@@ -935,11 +934,51 @@ install_debian_systemd() {
     done < <(grep -E '^Package:' "${SOURCE_DIR}/debian/control" | cut -d ':' -f 2)
 }
 
+install_suse_systemd() {
+    local testsdir=/usr/lib/systemd/tests
+    local pkgs
+
+    dinfo "Install SUSE systemd"
+
+    pkgs=(
+        systemd udev
+        systemd-container
+        systemd-coredump
+        systemd-experimental
+        systemd-journal-remote
+        systemd-portable
+    )
+
+    for p in ${pkgs[*]}; do
```

Comment:
```suggestion
    for p in "${pkgs[@]}"; do
```

Reply:
none of the package names includes whitespaces so I'm not sure to see the gain here.

Reply:
The main point here is that it makes shellcheck happy (once I finally finish shellchecking the whole tree). And, of course, it makes the code more robust.

Reply:
More robust in which way ? as I said package names can't contain any whitespace...

Reply:
We want to eventually shellcheck all the scripts, so please just run shellcheck and fix these issues